packages/client-sdk-web/src/utils/web-client-utils.ts

```diff
@@ -2,11 +2,13 @@ import {CredentialProvider} from '@gomomento/sdk-core';
 
 export function convertToB64String(v: string | Uint8Array): string {
   if (typeof v === 'string') {
-    return btoa(v);
+    const utf8Bytes = new TextEncoder().encode(v);
+    const binaryString = String.fromCharCode(...utf8Bytes);
+    return btoa(binaryString);
```

Comment:
this looks a lot cleaner/safer than the previous versions. i'm okay with this unless you find something better.